Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,28 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: $foot-height;
+  justify-content: center;
+  left: 0;
+  position: absolute;
```

Comment:
you don't need to set this  with position absolute, let it just place at the bottom automatically.